tests/components/asuswrt/test_sensor.py

```diff
@@ -226,6 +226,29 @@ async def test_loadavg_sensors_http(hass: HomeAssistant, connect_http) -> None:
     await _test_loadavg_sensors(hass, CONFIG_DATA_HTTP)
 
 
+async def test_loadavg_sensors_unaivalable_http(
+    hass: HomeAssistant, connect_http
+) -> None:
+    """Test load average sensors no available using http."""
+    config_entry, sensor_prefix = _setup_entry(hass, CONFIG_DATA_HTTP, SENSORS_LOAD_AVG)
+    config_entry.add_to_hass(hass)
+
+    connect_http.return_value.async_get_loadavg.side_effect = (
+        AsusWrtNotAvailableInfoError
+    )
+
+    # initial devices setup
+    assert await hass.config_entries.async_setup(config_entry.entry_id)
+    await hass.async_block_till_done()
+    async_fire_time_changed(hass, utcnow() + timedelta(seconds=30))
+    await hass.async_block_till_done()
+
+    # assert temperature sensor available
```

Comment:
Comment is wrong probably a copy-paste leftover

Reply:
Oh yes, corrected